system-addon/lib/TopSitesFeed.jsm

```diff
@@ -22,8 +23,11 @@ const DEFAULT_TOP_SITES = [];
 this.TopSitesFeed = class TopSitesFeed {
   constructor() {
     this.lastUpdated = 0;
+    this._tippyTopProvider = new TippyTopProvider();
```

Comment:
@k88hudson any thoughts on having this `TippyTopProvider` vs a `TippyTopFeed`? If we do want a feed, we should just refactor it later. I could see the feed dispatching a mapping of domains to urls, but then again, would we want all that data going to stores and the content process?